api/badgesize.ts

```diff
@@ -7,29 +7,32 @@ export default createBadgenHandler({
     '/badgesize/normal/amio/emoji.json/master/emoji-compact.json': 'normal size',
     '/badgesize/brotli/amio/emoji.json/master/emoji-compact.json': 'brotli size',
     '/badgesize/gzip/amio/emoji.json/master/emoji-compact.json': 'gzip size',
-    '/badgesize/normal/https/unpkg.com/snarkdown/dist/snarkdown.js': 'arbitrary url',
+    '/badgesize/normal/file-url/https/unpkg.com/snarkdown/dist/snarkdown.js': 'arbitrary url',
+    '/badgesize/normal/file-url/unpkg.com/snarkdown/dist/snarkdown.js': 'arbitrary url',
   },
   handlers: {
-    '/badgesize/:topic/:path+': handler
+    '/badgesize/:topic/file-url/:protocol<https?:?>/:hostname/:pathname+': urlHandler,
+    '/badgesize/:topic/file-url/:hostname/:pathname+': urlHandler,
+    '/badgesize/:topic/:protocol<https?:?>/:hostname/:pathname+': urlHandler,
```

Comment:
The idea here is to continue to support bare URL as param but favor `/file-url` because there is already `/metadata-url` for the Maven service.